### PR TITLE
feat: add analyze profile pass-through command"

### PR DESCRIPTION
**Pull request by gregmagolan**
- chore: sort plugins list in root.go
- feat: add analyze-profile pass-through command

Follow up to https://github.com/aspect-build/aspect-cli/pull/261. Look there for more context.

### CODE REVIEW

File: cmd/aspect/analyzeprofile/analyzeprofile.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	cmd := &cobra.Command{
+		Use:   "analyze-profile",
+		Short: "Analyzes build profile data.",
+		Long: `Analyzes build profile data for the given profile data files.
+
+Analyzes each specified profile data file and prints the results.  The
+input files must have been produced by the 'bazel build
+--profile=file' command.
+
+By default, a summary of the analysis is printed.  For post-processing
```
Comment (alexeagle) [inline cmd/aspect/analyzeprofile/analyzeprofile.go:42]: while we are here - I could imagine `analyze-profile` being amazingly more helpful. This is where we should insert a bunch of our "we understand what the problem is". Maybe worth brainstorming that into some issues

**Review by alexeagle — APPROVED** [review]

**Comment by alexeagle** [comment]
Can we amend the documentation to also fill in the "how do I get one of these profiles"?

With no arguments, could we analyze the "previous profile" - whatever is in `$(bazel info output_base)/command.profile.gz` ?

**Comment by gregmagolan** [comment]
> Can we amend the documentation to also fill in the "how do I get one of these profiles"?

sgtm. I can add a note to the docstring
 
> With no arguments, could we analyze the "previous profile" - whatever is in `$(bazel info output_base)/command.profile.gz` ?

lets leave anything that isn't pass-through behaviour to follow-up; will keep those changes isolated and be easier to review

**Comment by gregmagolan** [comment]
Hmmm. The current docstring from Bazel already has a note that a profile can be generated with `bazel build
--profile=file`. In the interest of time, I'll leave that and we can improve on it in a follow-up.

**Pull request merged**